pkg/controller/servingcert/controller/secret_creating_controller.go

```diff
@@ -333,17 +333,14 @@ func MakeServingCert(dnsSuffix string, ca *crypto.CA, intermediateCACert *x509.C
 	if err != nil {
 		return nil, err
 	}
-	if len(servingCert.Certs) > 1 {
-		// Only include the generated cert. The issuing cert will be provided to
-		// clients via the ca bundle.
-		servingCert.Certs = servingCert.Certs[:1]
-	}
+
```

Comment:
How about we keep the old behavior for `AlphaServingCertSecretAnnotation` (I think that was your original proposal), have the newer one for `beta` version of the annotation, then claim the `alpha` annotation as deprecated for 4.4 and remove it completely in 4.5? That should effectively move all the users who misuse this bundle to do the right thing in 4.5.

Reply:
That was what this PR originally proposed, but @sttts convinced me that changing the behavior of the beta annotation constituted a breaking change with a potential customer impact. I don't think the current behavior justifies a breaking change. As per the comment I added to the api file, I think it is preferable to fix the behavior for a GA annotation rather than introducing a breaking change.

Reply:
Would it make sense to make betav2 annotation with the more correct behavior, then? I don't think we would like to leap as much as doing a breaking change from beta to GA, in case we missed something else.

Reply:
I don't think there's value in worrying about fixing this behavior before GA.  And anyone using the serving cert as a ca bundle will not get the benefit of trust-maintaining rotation, so that ill-advised usage is likely to end even without a GA annotation.

Reply:
/lgtm
Fair enough for me